httpheader.php

```diff
@@ -2,12 +2,18 @@
 /**
  * HttpHeader Plugin
  *
- * @copyright  Copyright (C) 2017 - 2018 Tobias Zulauf All rights reserved.
+ * @copyright  Copyright (C) 2017 - 2019 Tobias Zulauf All rights reserved.
  * @license    http://www.gnu.org/licenses/gpl-2.0.txt GNU General Public License Version 2 or Later
  */
 
 defined('_JEXEC') or die;
 
+use Joomla\CMS\Application\CMSApplication;
+use Joomla\CMS\Language\Text;#
```

Comment:
Remove trailing `#`

Reply:
Thanks just did that change in my development branch. It will be fixed by the next commit in the comming days as I'm about to allocate time to adress a bruch of things that @yvesh already reported to me in the next week. Thanks for your report @ReLater

Reply:
resolved now